cmd/ferretdb/main.go

```diff
@@ -204,9 +214,86 @@ var (
 
 func main() {
 	setCLIPlugins()
-	kong.Parse(&cli, kongOptions...)
+	ctx := kong.Parse(&cli, kongOptions...)
+
+	switch ctx.Command() {
+	case "run":
+		run()
+	case "ping":
+		ping()
+	default:
+		panic("unknown sub-command")
+	}
+}
+
+// ping creates connection to FerretDB instance specified by the flags, and runs `ping` command against it.
+func ping() {
+	logger := setupLogger(setupState(), cli.Log.Format)
+	l := logger.Sugar()
+
+	checkFlags(logger)
+
+	if cli.Setup.Database == "" {
+		return
```

Comment:
Returning without any log message is misleading, this makes us assume ping was successful. By default `cli.Setup.Database` is empty string.

Reply:
This behavior is improved in the next PR that focuses on logging improvements #4364.